harvard-lts/CURIOSity
# Use redis dev

**Use redis config in dev**
* * *

# What does this Pull Request do?
Need to have redis on dev servers for replicas.

# How should this be tested?

Make sure everything comes up -- you shouldn't actually see any different behavior

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
